p0_user_stories.md

```diff
@@ -6,13 +6,13 @@ Taken from https://docs.google.com/document/d/10t4q5XO1ED2PnK3ishn4y3G4Tma7uMYge
 
 ## tier-1
 
-- I want 2 apps in different namespaces to connect, but can't read (or write) the labels for those namespaces from my Kubernetes client.   I cannot select pods for this because I don't know all the labels and names of pod/services that i want to contact ahead of time. 
+- I want 2 apps in different namespaces to connect, but can't read (or write) the labels for those namespaces from my Kubernetes client.   I cannot select pods for this because I don't know all the labels and names of pod/services that I want to contact ahead of time. 
 
-- I want different pods (with potentially different labels) that fulfill a service to be able talk to each other, without talking to other services...  but don't know the labels on these services.
+- I want different pods (with potentially different labels) that fulfill a service to be able talk to each other, without talking to other services but I don't know the labels on these services.
```

Comment:
Hum, thinking loud here, I think there's a point here that namespaces name are immutable, while labels are not. I agree labels are most flexible and the best way, but also one (with the right permissions) could change them and break things?

As an example, imagine I want to create an egress rule that allows me to connect to a destination namespace (cant remember if egress allows the ``to`` field to a namespace), but might this be a case? 

Also looking into the original issue (https://github.com/kubernetes/kubernetes/issues/88253) the description is:

"

Why is this needed:
While matching things like pods etc by label is certainly worthwhile, when matching a namespace I suspect the majority of the time you only want to match a single namespace. It been great match against the name rather than just a label. I suspect most people don't think to add labels to the namespaces."

Reply:
needs verify #15